core/src/main/java/tech/pegasys/web3signer/core/Runner.java

```diff
@@ -161,8 +162,26 @@ public void run() {
 
       registerHealthCheckProcedure(DEFAULT_CHECK, promise -> promise.complete(Status.OK()));
 
+      // Create shared worker executor for reload operations with generous timeout
+      // Pool size of 1 ensures reloads are serialized (only one reload at a time)
+      final WorkerExecutor reloadWorkerExecutor =
+          vertx.createSharedWorkerExecutor(
+              "web3signer-reload-pool",
+              1, // pool size
+              30, // max execute time
```

Comment:
This is a really long timeout. Does it need to be this long? Make it configurable?